euphonic/qpoint_phonon_modes.py

```diff
@@ -340,10 +340,10 @@ def calculate_debye_waller(
         .. math::
 
           W^{\\alpha\\beta}_{\\kappa} =
-          \\frac{\\hbar}{4M_{\\kappa}\\sum\\limits_{\mathbf{q}}{weight_\mathbf{q}}}
-          \\sum\\limits_{\mathbf{q}\\nu \in{BZ}}weight_\mathbf{q}\\frac{e_{\mathbf{q}\\nu\\kappa\\alpha}e^{*}_{\mathbf{q}\\nu\\kappa\\beta}}
+          \\frac{\\hbar}{4M_{\\kappa}\\sum\\limits_{\mathbf{q}}{\\mathrm{weight}_\mathbf{q}}}
+          \\sum\\limits_{\mathbf{q}\\nu \in{BZ}}\\mathrm{weight}_\mathbf{q}\\frac{e_{\mathbf{q}\\nu\\kappa\\alpha}e^{*}_{\mathbf{q}\\nu\\kappa\\beta}}
           {\\omega_{\mathbf{q}\\nu}}
-          coth(\\frac{\\hbar\\omega_{\mathbf{q}\\nu}}{2k_BT})
+          \\mathrm{coth}(\\frac{\\hbar\\omega_{\mathbf{q}\\nu}}{2k_BT})
```

Comment:
Looks good! The text below is now a bit out of sync: need to update epsilon -> e [is that _really_ a good idea? We haven't update Debye-Waller to "exp" if we are going that way to match the paper], bold q in ω<sub>**q**ν</sub>, upright "weight" with bold q.

The coth section would look better (and match paper) with big parens:
```
\\mathrm{coth}\\left(\\frac{\\hbar\\omega_{\mathbf{q}\\nu}}{2k_BT}\\right)
```